### ISSUE TITLE
empty: add simplify Zero and Empty compared to []bytes converted into string

### ISSUE THREAD

**Issue by ccoVeille** [Antonboom/testifylint#112]
https://github.com/search?q=language%3Ago+%22assert.Empty%28t%2C+string%28%22&type=code
https://github.com/search?q=language%3Ago+%22assert.Zero%28t%2C+string%28%22&type=code

Here are some examples

https://github.com/jroehl/go-suitesync/blob/01764321396a29047cc96ba2c0680c5291993c8c/lib/helper_test.go#L195

```go
        assert.Zero(t, string(res))
```

https://github.com/testmeifyoucan/schreder/blob/fd29194cd8aacd59e99e2ee91fe36305a7e06fa1/testrunner.go#L203

```go
        return assert.Empty(t, string(responseBody), "expected empty response")
```

They should be replaced by calling empty

**Comment by ccoVeille** [comment]
Duplicated by #119
